src/components/info-reveal/styles.css

```diff
@@ -0,0 +1,103 @@
+:host {
+  --z-info-reveal-theme--surface: var(--color-black);
+  --z-info-reveal-theme--text: var(--color-white);
+  --z-info-reveal-shadow: var(--shadow-2);
```

Comment:
c'è un problema sull'ombreggiatura: quando si apre per la prima volta il componente, l'ombreggiatura mantiene le dimensioni che aveva in precedenza, quindi se avevo solo l'icona, l'ombreggiatura in basso la vedo presente solo sui primi 32 pixel corrispondendi alla larghezza del componente da chiuso

Reply:
giusto, sistemate ;)